Add a metaTitle and a metaDescription to every doc pages (#233)
strapi/documentation: docs/user-docs/latest/settings/managing-global-settings.md

---
title: Global Settings - Strapi User Guide

> description: Learn how you can manage the global settings of your Strapi application in the Admin UI.

Reviewer: ```suggestion
description: Instructions to manage the global settings of a Strapi application in the admin panel.
```
